packages/model/src/user.ts

```diff
@@ -142,7 +142,7 @@ export interface UserPatchRequest {
   skillsDescription?: string;
   questions?: string[];
   teams?: Pick<UserTeam, 'id' | 'approach' | 'responsibilities'>[];
-  social?: UserSocialLinks;
+  social?: Omit<UserSocialLinks, 'orcid'>;
```

Comment:
why dont you remove orcid from the `UserSocialLinks` instead? I don't think it's actually being used anywhere

Reply:
Because it is being used

Reply:
where?

Reply:
Social Icons & Social modal & fixtures tests etc

Reply:
yeah i see but how does that make any sense - logically - if orcid isn't actually available there? like.. unless this is meant to be a hotfix i would go through those fixtures and remove all the references

Reply:
It was supposed to move into socialLinks. It is available there to read but not to write it would seem